### PR TITLE
Bump amqp-client from 4.12.0 to 5.14.2

### PR DESCRIPTION
**Pull request by t-h-e**
Replace QueueingConsumer with DeliveryCallback, as QueueingConsumer has been removed with version 5 of amqp-client

### CODE REVIEW

File: src/main/java/com/zeroclue/jmeter/protocol/amqp/AMQPConsumer.java [hunk 1]
```diff
@@ -72,11 +84,12 @@ public SampleResult sample(Entry entry) {
...
             if (consumer == null) {
                 log.info("Creating consumer");
-                consumer = new QueueingConsumer(channel);
+                response = new LinkedBlockingQueue<>(1);
+                consumer = (consumerTag, delivery) -> response.offer(delivery);
             }
             if (consumerTag == null) {
                 log.info("Starting basic consumer");
-                consumerTag = channel.basicConsume(getQueue(), autoAck(), consumer);
+                consumerTag = channel.basicConsume(getQueue(), autoAck(), consumer, consumerTag  -> {});
```
Comment (t-h-e) [inline src/main/java/com/zeroclue/jmeter/protocol/amqp/AMQPConsumer.java:92]: This is the only part, I'm not completely sure about if the `CancelCallback` can simply be ignored.

**Comment by t-h-e** [comment]
@aliesbelik Thanks for keeping this plugin alive 😃

**Comment by t-h-e** [comment]
SonarCloud failed, but it seems unrelated to the actual change.

**Comment by aliesbelik** [comment]
@t-h-e Thank you for this update.
Fixes jlavallee#55 as well.

**Pull request merged**